File: Zad1-2-KonwerterTemperatur.cs
using System;
					
public class Program
{
	public static void Main()
	{
		Console.WriteLine("Konwerter Temperatur");
		Console.WriteLine("[c] Celsjusz -> Fahrenheit");
		Console.WriteLine("[f] Fahrenheit -> Celsjusz");
		Console.WriteLine("Wybierz kierunek konwersji: ");
		string choice = Console.ReadLine();
		
		Console.WriteLine("Podaj temperaturę: ");
		double temp = Convert.ToDouble(Console.ReadLine());
		
		if (choice == "f")
		{
			double temp2 = (temp - 32.0) / 1.8;
			Console.WriteLine($"{temp}°F = {temp2}°C");
		}
		else if (choice == "c")
		{
			double temp2 = temp * 1.8 + 32.0;
			Console.WriteLine($"{temp}°C = {temp2}°F");
		}
		else
		{
			Console.WriteLine("Nieprawidłowy wybór.");
		}
	}
}

File: zad1_2_konwerter_temperatur.py
def zad2():
    print("Konwerter Temperatur")
    print("[c] Celsjusz -> Fahrenheit")
    print("[f] Fahrenheit -> Celsjusz")
    choice = input("Wybierz kierunek konwersji:")
    
    if choice not in ["f", "c"]:
        print("Nieprawidlowy wybór")
        return
    temp = int(input("Podaj temperature: "))

    if choice == "f":
        temp2 = (temp - 32) / 1.8
        print(f"{temp}°F = {temp2}°C")
    else:
        temp2 = temp * 1.8 + 32
        print(f"{temp}°C = {temp2}°F")

zad2()
